tests/SideBySide/AppConfig.cs

```diff
@@ -19,18 +19,17 @@ public static class AppConfig
 				["Data:SupportsJson"] = "false",
 			};
 
-		private static string CodeRootPath = GetCodeRootPath();
+		public static string CertsPath => Config.GetValue<string>("Data:CertificatesPath")
```

Comment:
I think it handles the conversion of `../` to `..\` on Windows.  Change all of the CertFile references to `../` in the `config.json` files for consistency and this should be good.